pymilvus_export_collection_europeana.py

```diff
@@ -0,0 +1,35 @@
+import json
+from pymilvus import Collection, connections
+
+milvus_host="localhost"
+milvus_port="19530"
+milvus_collection="apr2024"
+number_or_entries_to_return_per_call=1000
+milvus_collection_fields_to_return=["about","vector"]
```

Comment:
shouldn't be only vector by default? The experments don't use the record id, but we might need it for data migration (from older to newer milvus versions)